test/source/tests/compose.ts

```diff
@@ -1485,6 +1485,29 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       await composePage.waitAny('@password-or-pubkey-container', { visible: false });
     }));
 
+    ava.default('attester should understand more than one pub key', testWithBrowser('ci.tests.gmail', async (t, browser) => {
```

Comment:
The title should read something like `attester client should...` -- we're testing reading multiple keys from the attester.